ipam/pool.go

```diff
@@ -216,6 +220,9 @@ func (as *addressSpace) merge(newas *addressSpace) {
 			for ak, av := range pv.Addresses {
 				if av.epoch == as.epoch || av.InUse {
 					// Pool has at least one valid or in-use address.
+					if av.epoch != as.epoch {
+						av.Unhealthy = true
+					}
```

Comment:
this is not right. we have to set pv.epoch = as.epoch if av.InUse is set. So do you want me to add that in else if av.InUse case